src/components/quests/QuestPageFooter/QuestPageFooter.tsx

```diff
@@ -0,0 +1,14 @@
+import type { Doc } from "@convex/_generated/dataModel";
+import { QuestUpdated } from "../QuestUpdated/QuestUpdated";
+
+interface QuestPageFooterProps {
+  quest: Doc<"quests">;
+}
+
+export function QuestPageFooter({ quest }: QuestPageFooterProps) {
+  return (
+    <div className="border-t border-gray-dim py-4">
```

Comment:
```suggestion
    <footer className="border-t border-gray-dim py-4">
```
Lil semantic improvement